src/server/api/routers/commission.ts

```
@@ -145,20 +145,20 @@ export const commission_router = createTRPCRouter({
 
             // Update algolia
-            await update_index('commissions', {
+            await updateIndex('commissions', {
                 objectID: input.id,
                 title: updated_data.title,
                 price: updated_data.price
                     ? formatToCurrency(updated_data.price / 100)
                     : undefined,
                 description: updated_data.description!,
-                featured_image: get_ut_url(updated_data.images![0]!.ut_key),
+                featured_image: getUTUrl(updated_data.images![0]!.ut_key),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)